src/leap/mail/imap/account.py

```diff
@@ -49,6 +49,16 @@ def _debugProfiling(result, cmdname, start):
 # Soledad IMAP Account
 #######################################
 
+class IIMAPAccount(Interface):
+    """
+    The abstraction describing an IMAPAccount.
+    """
```

Comment:
I guess this is somehow related to `imap4.IAccount`, isn't it? I'm not sure if `zope.interface` has something to state this relation, but maybe we can mention it in the docstring.